src/components/Dashboard/DashboardPage.tsx

```diff
@@ -112,6 +184,61 @@ export function DashboardPage(props: DashboardPageProps) {
                     />
                 </div>
             </div>
+            <div
+                className={fr.cx(
+                    "fr-grid-row",
+                    "fr-grid-row--gutters",
+                    "fr-mt-6w"
```

Comment:
```suggestion
                    "fr-mt-6w",
                    "fr-px-6w"
```

À défaut d'avoir la pleine largeur, au moins un peu de padding sur les côtés